src/containers/connected-login-modal/connected-login-modal.js

```diff
@@ -1,26 +1,18 @@
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as LoginModalActions from '../../actions/login-modal';
 import loginModalComponent from '../../components/login-modal';
-import Actions from '../../reducers/actions';
 
 const mapStateToProps = state => ({
   isLoginModalOpen: state.login.isLoginModalOpen,
 });
-const mapDispatchToProps = dispatch => ({
-  closeLoginModal: () => {
-    const action = {
-      type: Actions.login.closeLoginModal,
-    };
-
-    dispatch(action);
-  },
-  swapModal: () => {
-    const action = {
-      type: Actions.login.swapModal,
-    };
-
-    dispatch(action);
-  },
-});
+const mapDispatchToProps = dispatch =>
+  bindActionCreators(
+    {
+      ...LoginModalActions,
```

Comment:
I'd like to stick to a casing standard of only classes are capitalized. This shouldn't be.

Reply:
i changed it to loginModalActions if that's what you were referring to. and did the same for the rest of the actions.